release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -1058,16 +1058,21 @@ In the following tables, features are marked with the following statuses:
 |Technology Preview
 |Technology Preview
 
-|Admin Network Policy (`AdminNetworkPolicy`)
+|`AdminNetworkPolicy` (ANP)
 |Not Available
 |Technology Preview
 |Technology Preview
 
-|IPsec external traffic (north-south)
+|`BaselineAdminNetworkPolicy` (BANP)
```

Comment:
ah thanks for the preview! I see it now.. BANP and ANP are not two different features, tl;dr we only need to list ANP in the list, ANP is the TP Feature which offers two APIs ANP/BANP.

Reply:
I'll fix in a new PR. Thank you!

Reply:
I think we might need to standardize how we are talking about `AdminNetworkPolicy` the API and Admin Network Policy the feature. @tssurya cc @ahardin-rh

Reply:
@JoeAldinger I agree. We want to be as clear as possible to avoid any confusion. I'd love to see us standardize this in 4.15 if possible.